volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -65,10 +201,141 @@ def create_callback_table(
+    ) -> Tuple[
+        Union[str, interfaces.renderers.BaseAbsentValue],
+        int,
+        Union[str, interfaces.renderers.BaseAbsentValue],
+    ]:
+        symbol_table = memory_object.get_symbol_table_name()
+        type_name = memory_object.vol.type_name
+
+        if isinstance(memory_object, callbacks._SHUTDOWN_PACKET):
+            callback_type = "IoRegisterShutdownNotification"
+
+            try:
+                driver = memory_object.DeviceObject.dereference().DriverObject
+                index = driverirp.MAJOR_FUNCTIONS.index("IRP_MJ_SHUTDOWN")
+                callback_address = driver.MajorFunction[index]
+                details = driver.DriverName.String or renderers.UnparsableValue()
+            except exceptions.InvalidAddressException:
+                callback_address = memory_object.vol.offset
+                details = renderers.NotApplicableValue()
+
+        elif type_name == symbol_table + constants.BANG + "_NOTIFICATION_PACKET":
+            callback_type = "IoRegisterFsRegistrationChange"
+            callback_address = memory_object.NotificationRoutine
+            details = renderers.NotApplicableValue()
+
+        elif type_name == symbol_table + constants.BANG + "_NOTIFY_ENTRY_HEADER":
+            driver = (
+                memory_object.DriverObject.dereference()
+                if memory_object.DriverObject.is_readable()
+                else None
+            )
+
+            if driver:
+                # Instantiate an object header for the driver name
+                header = driver.get_object_header()
+                try:
+                    if header.get_object_type(type_map) == "Driver":
+                        # Grab the object name
+                        details = header.NameInfo.Name.String
+                    else:
+                        details = renderers.NotApplicableValue()
+                except exceptions.InvalidAddressException:
+                    details = renderers.UnreadableValue()
+            else:
+                details = renderers.UnreadableValue()
+
+            callback_type = (
+                memory_object.EventCategory.description
+                if memory_object.EventCategory.is_valid_choice
+                else renderers.UnparsableValue()
+            )
+
+            callback_address = memory_object.CallbackRoutine
+
+        elif type_name == symbol_table + constants.BANG + "_GENERIC_CALLBACK":
+            callback_type = "GenericKernelCallback"
+            callback_address = memory_object.Callback
+            details = renderers.NotApplicableValue()
+
+        elif type_name == symbol_table + constants.BANG + "_DBGPRINT_CALLBACK":
+            callback_type = "DbgSetDebugPrintCallback"
+            callback_address = memory_object.Function
+            details = renderers.NotApplicableValue()
+
+        else:
+            raise ValueError("Unexpected object type %s" % type_name)
```

Comment:
Could we use `f"Unexpected object type {type_name}"` for consistency and readability please?